src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.ts

```diff
@@ -486,16 +533,21 @@ export class NfsFormComponent extends CdForm implements OnInit {
   }
 
   private generatePseudo() {
-    let newPseudo = this.nfsForm.getValue('pseudo');
-    if (this.nfsForm.get('pseudo') && !this.nfsForm.get('pseudo').dirty) {
-      newPseudo = undefined;
-      if (this.storageBackend === 'CEPH') {
-        newPseudo = '/cephfs';
-        if (_.isString(this.nfsForm.getValue('path'))) {
-          newPseudo += this.nfsForm.getValue('path');
-        }
+    const pseudoControl = this.nfsForm.get('pseudo');
+    let newPseudo =
+      pseudoControl && pseudoControl.dirty ? this.nfsForm.getValue('pseudo') : undefined;
+
+    if (!newPseudo) {
```

Comment:
```suggestion
    const pseudoControl = this.nfsForm.get('pseudo');
    if (!(pseudoControl?.dirty && pseudoControl?.value) {
```

Reply:
We anyways need newPseudo value later, so I think it doesn't hurt declaring it here already